src/entities/MovieDetails/MovieDetails.jsx

```diff
@@ -1,9 +1,50 @@
 import "./MovieDetails.css";
 
-export const MovieDetails = ({ movie, handleMovieViewClosed }) => {
+import {
+  useLoaderData,
+  useSearchParams,
+  createSearchParams,
+  useNavigate,
+} from "react-router-dom";
 
-  const { imgUrl, movieName, releaseYear, rating, duration, description } =
-    movie;
+export const MovieDetails = ({
+  movie = {
+    movieName: "La La Land",
+    releaseYear: 2016 - 12 - 29,
+    imgUrl: "https://via.placeholder.com/300/92c952",
+    duration: "128",
+    rating: 7.9,
+    description:
+      "Mia, an aspiring actress, serves lattes to movie stars in between auditions and Sebastian, a jazz musician, scrapes by playing cocktail party gigs in dingy bars, but as success mounts they are faced with decisions that begin to fray the fragile fabric of their love affair, and the dreams they worked so hard to maintain in each other threaten to rip them apart.",
+  },
+}) => {
+  const [searchParams] = useSearchParams();
+  const navigate = useNavigate();
```

Comment:
https://reactrouter.com/en/main/hooks/use-navigate

<img width="604" alt="image" src="https://github.com/AnnaMartis/Epam-project/assets/36797396/ed9fa4b8-f2c4-4e82-a4ca-36ad4975f3af">

Reply:
don't they mean inside loader files ?

Reply:
using `redirect` inside `loader` doesn't mean that